components/desktop/hexchat/Makefile

```diff
@@ -56,18 +57,15 @@ CONFIGURE_OPTIONS += -Dwith-python=python-3.9
 CONFIGURE_OPTIONS += -Dwith-lua=false
 CONFIGURE_ENV += AR="/usr/gnu/bin/ar"
 
-REQUIRED_PACKAGES += runtime/perl-534
-REQUIRED_PACKAGES += runtime/python-39
-
 # Auto-generated dependencies
+PERL_REQUIRED_PACKAGES += runtime/perl
 REQUIRED_PACKAGES += library/desktop/gdk-pixbuf
 REQUIRED_PACKAGES += library/desktop/gtk2
 REQUIRED_PACKAGES += library/desktop/pango
 REQUIRED_PACKAGES += library/desktop/xdg/libcanberra
 REQUIRED_PACKAGES += library/glib2
-REQUIRED_PACKAGES += library/libnotify
-REQUIRED_PACKAGES += library/libproxy
 REQUIRED_PACKAGES += library/security/openssl-11
+REQUIRED_PACKAGES += runtime/python-39
```

Comment:
Was this generated using the latest oi-userland source tree?  If not, then please rebase and run `gmake REQUIRED_PACKAGES` again.  Thanks.

Reply:
You are right, using `PYTHON_REQUIRED_PACKAGES += runtime/python` now. That's even better.

Reply:
Thanks.  You should run `gmake publish` too to update the `pkg5` file.  Sorry for not mentioning this earlier, I forgot.